### ISSUE TITLE
Antd style problems in dark mode 

### ISSUE THREAD

**Issue by sedatkimya** [hongfaqiu/nextjs13-with-antd-tailwindcss#3]
First of all, congratulations for this great job.
There are problems with antd components in dark mode. What could be the reason for this?
Examples:
Primary button is not shown in blue.
The background is not visible in the switch component (if not selected).
<img width="194" alt="image" src="https://user-images.githubusercontent.com/45509895/229965534-8f9f3140-08da-4a21-9c21-b7385910863b.png">

Thanks.

**Comment by hongfaqiu** [comment]
Thank you for your discovery. This problem was caused by Tailwind's preflight CSS. I have set the hashPriority property of the StyleProvider to "high", but it seems to not work in dark mode. To resolve this, I copied the base CSS from Tailwind to the global.css file and disabled the preflight configuration. This has resolved the conflict and it now works with no issues (both with the no-css-in-js branch).
